index.html

```diff
@@ -4,6 +4,9 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+<h1>
+  <img src "https://media.giphy.com/media/11Lz1Y4n1f2j96/giphy.gif">
```

Comment:
- Did you check that the image works / open it in your browser?
- Are you sure you want to nest an image (`img`) inside of a heading (`h1`)?